### ISSUE TITLE
null view state

### ISSUE THREAD

**Issue by fahimk** [sockeqwe/mosby#26]
Seeing this issue sometimes. All my createViewState functions have been implemented creating new instances of states.

```java
java.lang.NullPointerException: ViewState is null! That's not allowed
       at com.hannesdorfmann.mosby.mvp.viewstate.ViewStateManager.saveInstanceState(ViewStateManager.java:103)
       at com.hannesdorfmann.mosby.mvp.viewstate.MvpViewStateFragment.saveViewStateInstanceState(MvpViewStateFragment.java:79)
       at com.hannesdorfmann.mosby.mvp.viewstate.MvpViewStateFragment.onSaveInstanceState(MvpViewStateFragment.java:70)
       at android.support.v4.app.Fragment.performSaveInstanceState(Fragment.java:1936)
       at android.support.v4.app.FragmentManagerImpl.saveFragmentBasicState(FragmentManager.java:1654)
       at android.support.v4.app.FragmentManagerImpl.saveAllState(FragmentManager.java:1722)
       at android.support.v4.app.FragmentActivity.onSaveInstanceState(FragmentActivity.java:527)```

**Comment by sockeqwe** [comment]
Are you using Fragments with Backstack?

**Comment by fahimk** [comment]
yes, they are non retained fragments that are added to backstack

**Comment by sockeqwe** [comment]
I see, I have already fixed that problem. The fix will be part of mosby 1.1 which will be released soon.

**Comment by sockeqwe** [comment]
@fahimk I have released a SNAPSHOT that should contain a bugfix for that. Can you please try it:
```groovy
allprojects {
  repositories {
    jcenter()
    maven{
      url 'https://oss.sonatype.org/content/repositories/snapshots/'
    }
}
```

Version is `1.1.0-SNAPSHOT`
